### PR TITLE
Feature/home page

### PR DESCRIPTION
**Pull request by teamchz**

### CODE REVIEW

**Comment by aofattaporn** [comment]
27 file เอง 🥹

**Comment by aofattaporn** [comment]
don't forget to review code @sura1542  @thunyathep  @Balloii

**Review by aofattaporn — COMMENTED** [review]
Can you clean code by "**remove this comment**"

File: lib/App.dart [hunk 1]
```diff
@@ -25,10 +26,18 @@ class App extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
+      // theme: ThemeData(
```
Comment (aofattaporn) [inline lib/App.dart]: Can you clean code by "remove this comment"

File: lib/src/AppRoute.dart [hunk 2]
```diff
@@ -10,7 +10,7 @@ class AppRoute {
 
   final _route = <String, WidgetBuilder>{
     // view_post: (context) => ViewPost(title: "ViewPost"),
```
Comment (aofattaporn) [inline lib/src/AppRoute.dart]: - if which line didn't use, pls remove its 
- and I don't understand what to do in this line, pls to comment what is it to do for

File: lib/src/data/respository/homepage.dart [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  Future<String> getHomePage() async {
+    final response = await http.get(Uri.parse(_baseUrl));
+    if (response.statusCode == 201) {
+      return "create data success";
+    } else {
+      return "something fail.";
+    }
+  }
+
+  // Future<TestModel> getHomePageByCacheLib() async {
```
Comment (aofattaporn) [inline lib/src/data/respository/homepage.dart]: if not use, pls remove its

File: lib/src/screens/ProfilePage.dart [hunk 4]
```diff
@@ -22,17 +22,15 @@ class _ProfilePageState extends State<ProfilePage> {
   final List<PostData> _postData = [
```
Comment (aofattaporn) [inline lib/src/screens/ProfilePage.dart]: I'm not sure, where the variables do. 
if not use pls remove its.

**Review by aofattaporn — COMMENTED** [review]
Most of my review is about Comments that aren't understanding and Comments that should remove

File: lib/src/screens/ProfilePage.dart (hunk 4 above)
Comment (Balloii) [reply lib/src/screens/ProfilePage.dart:22]: Same here, Is the Postdata list has any variables inside?

**Review by Balloii — COMMENTED** [review]
Apart from what Aof already commented on, everything looks good to merge for me.

**Review by aofattaporn — APPROVED** [review]
LGTM

**Pull request merged**